packages/lua-language-server/build.sh

```diff
@@ -49,20 +50,12 @@ termux_step_make_install() {
 
 	cat > "${TERMUX_PREFIX}/bin/${TERMUX_PKG_NAME}" <<- EOF
 		#!${TERMUX_PREFIX}/bin/bash
+		TMPPATH="\$(mktemp -d "${TERMUX_PREFIX}/tmp/${TERMUX_PKG_NAME}.XXXX")"
 
-		# After action of termux-elf-cleaner lua-language-server's binary is unable to
-		# determine its version, so provide it manually.
-		if [ "\$1" = "--version" ]; then
-			echo "${TERMUX_PKG_NAME}: ${TERMUX_PKG_VERSION}"
-		else
-			TMPPATH=\$(mktemp -d "${TERMUX_PREFIX}/tmp/${TERMUX_PKG_NAME}.XXXX")
-
-			exec ${datadir}/bin/${TERMUX_PKG_NAME} \\
-				--logpath="\${TMPPATH}/log" \\
-				--metapath="\${TMPPATH}/meta" \\
-				"\${@}"
-		fi
-
+		exec ${datadir}/bin/${TERMUX_PKG_NAME} \\
+		\	--logpath="\${TMPPATH}/log" \\
```

Comment:
```suggestion
			--logpath="\${TMPPATH}/log" \\
```
Whoops forgot to remove those from testing.